releng/config-forker/main.go

```diff
@@ -351,6 +353,23 @@ func generateNameVariant(name, version string, generic bool) string {
 	return strings.ReplaceAll(name, "-master", suffix)
 }
 
+func generatePresubmitNameVariant(name, version string) string {
+	suffix := "-" + version
+	if !strings.HasSuffix(name, "-master") {
+		return name + suffix
+	}
+	return strings.ReplaceAll(name, "-master", suffix)
```

Comment:
I recommend making this a little helper function since it is used in three places, or making `"-master"` a global const.